ee/src/main/java/org/jboss/as/ee/concurrent/service/ElytronManagedThreadFactory.java

```diff
@@ -41,8 +41,14 @@
  */
 public class ElytronManagedThreadFactory extends ManagedThreadFactoryImpl {
 
+    /**
+     * the CL in context when instance was created, to be inherited by created threads
+     */
+    private final ClassLoader fccl;
+
     public ElytronManagedThreadFactory(String name, ContextServiceImpl contextService, int priority) {
         super(name, contextService, priority);
+        fccl = WildFlySecurityManager.getCurrentContextClassLoaderPrivileged();
```

Comment:
It probably makes sense to just use the `org.jboss.as.ee` module class loader here instead of hoping we get the right one.